src/GafferSceneUI/AttributeInspector.cpp

```diff
@@ -248,18 +248,36 @@ IECore::ConstObjectPtr AttributeInspector::value( const GafferScene::SceneAlgo::
 	return nullptr;
 }
 
-IECore::ConstObjectPtr AttributeInspector::fallbackValue( const GafferScene::SceneAlgo::History *history ) const
+IECore::ConstObjectPtr AttributeInspector::fallbackValue( const GafferScene::SceneAlgo::History *history, std::string &description ) const
 {
 	if( const auto inheritedAttribute = history->scene->fullAttributes( history->context->get<ScenePlug::ScenePath>( ScenePlug::scenePathContextName ) )->member( m_attribute ) )
 	{
+		ScenePlug::PathScope pathScope( history->context.get() );
+		pathScope.setCanceller( Context::current()->canceller() );
+
+		ScenePlug::ScenePath currentPath( history->context->get<ScenePlug::ScenePath>( ScenePlug::scenePathContextName ) );
```

Comment:
Should we pop one name off here (after checking we're not at the root), so we start at the parent location rather than the current location?

Reply:
Sounds good, it does make `fallbackValue()` a little more useful for the speculative "what will I end up with if I remove the attribute?" use-case. 1872db4